cob_monitoring/src/cpu_monitor.py

```diff
@@ -42,6 +45,14 @@ def __init__(self, hostname, diag_hostname):
         self._mem_warn = rospy.get_param('~mem_warn', 25)
         self._mem_error = rospy.get_param('~mem_error', 1)
 
+        self._check_thermal_throttling_events = rospy.get_param('~check_thermal_throttling_events', True)
```

Comment:
the other `check_*` params use `False` as default in order to not produce stale/error diagnostics for robots that do not want/have the respective tools setup (ipmi, netdata,...)...we should do the same for the new throttling and jitter diagnostics